src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationStatusValidatorImplTest.java

```diff
@@ -88,7 +88,7 @@ void test_validateAndReturnStatusList_invalid_status_check() {
     }
 
     void verifyException(String status, String message) {
-        assertThatThrownBy(() -> organisationStatusValidatorImpl.validateAndReturnStatusList(status))
+        Assertions.assertThatThrownBy(() -> organisationStatusValidatorImpl.validateAndReturnStatusList(status))
```

Comment:
Please revert back to the static import that was being used previously

Reply:
Fixed.